src/components/FeedbackWidget.tsx

```diff
@@ -220,8 +240,23 @@ const FeedbackWidget: React.FC<IProps> = ({ className }) => {
 
   return (
     <>
-      <FixedDot onClick={handleOpen} bottomOffset={bottomOffset} id="dot">
+      <FixedDot
+        onClick={handleOpen}
+        bottomOffset={bottomOffset}
+        isExpanded={isExpanded}
+        id="dot"
+      >
         <StyledFeedbackGlyph />
+        <Text
```

Comment:
I think we could wrap this `Text` with a transition like `ScaleFade` (https://chakra-ui.com/docs/components/transitions/usage#scalefade-transition) and delay it a bit.

IMO the animation should be:
1. Expand the button to the left (keeping the icon on the left)
2. Once the expand animation finish, we display the Text with a fade in or scale fade animation